bin/build-debian

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Can u rename to build-debian.sh? That's the convention everywhere. 

Also the sheshang should be: #!/usr/bin/env bash
It's more portable.